node/task.ts

```diff
@@ -1145,45 +1256,44 @@ function _legacyFindFiles_getMatchingItems(
  * Remove a path recursively with force
  * Returns whether it succeeds
  * 
- * @param     path     path to remove
+ * @param     inputPath     path to remove
  * @returns   void
  */
-export function rmRF(path: string): void {
-    debug('rm -rf ' + path);
-
-    // get the lstats in order to workaround a bug in shelljs@0.3.0 where symlinks
-    // with missing targets are not handled correctly by "rm('-rf', path)"
-    let lstats: fs.Stats;
-    try {
-        lstats = fs.lstatSync(path);
-    }
-    catch (err) {
-        // if you try to delete a file that doesn't exist, desired result is achieved
-        // other errors are valid
-        if (err.code == 'ENOENT') {
-            return;
+export function rmRF(inputPath: string): void {
+    debug('rm -rf ' + inputPath);
+    if (fs.existsSync(inputPath)) {
+        const inputStats: fs.Stats = fs.lstatSync(inputPath);
```

Comment:
Can this return null?

Reply:
Not if `inputPath` exists (which we've already checked).